src/components/data/data-utils.js

```diff
@@ -61,13 +62,19 @@ export const getDownloadCommand = (fileName, gsUri, accessUrl) => {
   if (gsUri) {
     return `gsutil cp ${gsUri} ${fileName || '.'}`
   }
+
+  if (azureUri) {
+    // TODO: Aaron
+    return `gsutil cp ${gsUri} ${fileName || '.'}`
+  }
 }
 
 export const getUserProjectForWorkspace = async workspace => (workspace && await canUseWorkspaceProject(workspace)) ?
-  workspace.workspace.googleProject :
+  isGoogleWorkspace(workspace) :
   requesterPaysProjectStore.get()
 
-const isUri = datum => _.startsWith('gs://', datum) || _.startsWith('dos://', datum) || _.startsWith('drs://', datum)
+const isUri = (datum, workspace) => (isGoogleWorkspace(workspace) &&
+  isGs(datum)) || isDrs(datum) || (isAzureWorkspace(workspace) && (isAzureUri(datum)))
```

Comment:
I think this change makes the function name misleading. A string could be a URI regardless of which type of workspace it's in. Is the new meaning of this something like `isViewableUri`?

Reply:
Yeah, it looks like the usage of this is:
```js
return isUri(datum, workspace) ? h(UriViewerLink, { uri: datum, workspace }) : stringDatum
```

I'd vote for changing the name of this function. Even without this change, it's confusing since a reader could reasonably assume `isUri('https://example.com/file.txt')` would return true. Not necessary to add in this PR though.

Reply:
Updated naming!